upt_macports/templates/base.Portfile

```diff
@@ -4,16 +4,20 @@ PortSystem          1.0
 {% block portgroup %}
 {% endblock %}
 
+# Portfile generated/maintained using the Universal Packaging Tool (upt)
+
 {% block nameversion %}
 {% endblock %}
 
-maintainers         nomaintainer
-description         {{ pkg.upt_pkg.summary }}
-long_description    ${description}
-
+# categories-append
```

Comment:
You could add "XXX" or "TODO" at the end of this line, so that it is clearly visible to the maintainer when they open the file. @reneeotten what do you think?

Reply:
I agree - we should signal in a clear way that a maintainer is supposed to do something about it. I think we should add one link to the MacPorts guide on top of that block in the Portfile, comment out lines where action is required, and add TODO at the end of the line. I have made some suggestions in the the base template.